modules/installing-ocp-agent.adoc

```diff
@@ -198,3 +233,37 @@ INFO To access the cluster as the system:admin user when using 'oc', run
 INFO     export KUBECONFIG=/home/core/installer/auth/kubeconfig
 INFO Access the OpenShift web-console here: https://console-openshift-console.apps.sno-cluster.test.example.com
 ----
+
+
+[NOTE]
+====
+If you are using the optional method of ZTP manifests, you can configure IP address endpoints for cluster nodes through the `AgentClusterInstall.yaml` file in three ways:
+
+* IPv4
+* IPv6
+* IPv4 and IPv6 in parallel (dual-stack)
+
+.Example of dual-stack networking
+[source,yaml]
+----
+apiVIP: 192.168.11.3
+ingressVIP: 192.168.11.4
+clusterDeploymentRef:
+  name: mycluster
+imageSetRef:
+  name: openshift-4.12
+networking:
+  clusterNetwork:
+  - cidr: 172.21.0.0/16
+    hostPrefix: 23
+  - cidr: fd02::/48
+    hostPrefix: 64
+  serviceNetwork:
```

Comment:
Nit - for consistency with above examples should move serviceNetwork after machineNetwork